src/nrepl/connectSequence.ts

```diff
@@ -34,7 +34,7 @@ interface CljsTypeConfig {
     isReadyToStartRegExp?: string | RegExp;
     openUrlRegExp?: string | RegExp;
     shouldOpenUrl?: boolean;
-    connectCode: string | Object;
+    connectCode: string | Record<string, string>;
```

Comment:
The type `Object` means "any non-nullish value" which is typically not what people mean. A more precise type gives us more safety here.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-types.md
https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492